### PR TITLE
DFPL-1493 CFV - Further application documents

### PR DESCRIPTION
**Pull request by so99y**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1493

### Change description ###
1. Enable CaseFileView to show the application documents uploaded.
2. Added migration script and rollback script.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json [hunk 1]
```diff
@@ -1594,5 +1594,47 @@
     "CaseFieldID": "previousProceedingListLA",
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanList",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "CRUD"
```
Comment (chak-shing-lo-justice) [inline ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json:1603]: wonder why cafcass need "CUD"? Think they can't upload any document, "R" should be enough for them?

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json [hunk 2]
```diff
@@ -6004,5 +5997,68 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanListLA",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanListCTSC",
```
Comment (chak-shing-lo-justice) [inline ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:6018]: looks like admin, gatekeeper and judges can't upload application document, only LA can upload them. 
Does that mean the CTSC list is always empty?

**Review by chak-shing-lo-justice — COMMENTED** [review]
LGTM, just two questions

File: ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json (hunk 1 above)
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json:1603]: I don't know. I copy it from `applicationDocuments`

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json (hunk 2 above)
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:6018]: Not sure if the new management document UI flow allows them to do so. Worth to keep them here for consistency. Perhaps I will implement this constraint in the new manage document flow.

**Review by chak-shing-lo-justice — APPROVED** [review]
lgtm

**Review by chak-shing-lo-justice — APPROVED** [review]
lgtm

**Pull request closed without merging**